index.bs

```diff
@@ -869,6 +869,7 @@ interface MLContext {
   undefined dispatch(MLGraph graph, MLNamedTensors inputs, MLNamedTensors outputs);
 
   Promise<MLTensor> createTensor(MLTensorDescriptor descriptor);
+  Promise<MLConstantTensor> createConstantTensor(MLOperandDescriptor descriptor, AllowSharedBufferSource inputData);
```

Comment:
nitpick: Wrap this to ~80 columns or it'll cause horizontal scrollbars to appear. The lint tool will flag it.

Reply:
Done.